AssertionError: Fatal: The coverage report file was not updated after the test command. file_mod_time_ms: 1716321605810, time_of_test_command: 1716321611774. False

AssertionError: Fatal: The coverage report file was not updated after the test command. file_mod_time_ms: 1716321605810, time_of_test_command: 1716321611774. False

Command run:

cover-agent --source-file-path "autogen_.py" --test-file-path "tests/test_autogen_.py" --code-coverage-report-path "coverage.cov" --test-command "pytest -s tests/test_autogen_.py" --test-command-dir "." --coverage-type "cobertura" --desired-coverage 80 --max-iterations 10

--- a/README.md
+++ b/README.md
@@ -51,6 +51,9 @@ This tool is part of a broader suite of utilities designed to automate the creat
 ### Requirements
 Before you begin, make sure you have the following:
 - `OPENAI_API_KEY` set in your environment variables, which is required for calling the OpenAI API.
+- Code Coverage tool: A Cobertura XML code coverage report is required for the tool to function correctly.
+  - For example, in Python one could use `pytest-cov`. Add the `--cov-report=xml` option when running Pytest.
+  - Note: We are actively working on adding more coverage types but please feel free to open a PR and contribute to `cover_agent/CoverageProcessor.py`
 
 If running directly from the repository you will also need:
 - Python installed on your system.

